src/govuk/template.njk

```diff
@@ -2,11 +2,14 @@
 {% from "./components/header/macro.njk" import govukHeader -%}
 {% from "./components/footer/macro.njk" import govukFooter -%}
 
+{%- set _rebrand = govukRebrand -%}
```

Comment:
Now we're not having to check if `govukRebrand` is callable everywhere I'm not sure there's much value in having this variable. Can we remove it and simplify?